README.md

```diff
@@ -30,6 +30,7 @@ Tarka UI Kit is a pre-built component library for building Flutter apps. It is b
 - TUIDraggableCard
 - TUIEmailField
 - TUIEmailSubjectField
+- TUIFilePicker
```

Comment:
Should it be in the component list? It is not designed yet.

Reply:
Yes I think we can drop this